src/app/store/reducers/tariff.reducer.ts

```diff
@@ -27,7 +27,13 @@ export const tariffReducer = createReducer(
       stateLocation.locationsDto.forEach((locationDto, locationDtoIndex) => {
         action.editedLocations.forEach((actionEditedLocation) => {
           if (locationDto.locationId === actionEditedLocation.locationId) {
-            stateLocations[stateLocationIndex].locationsDto[locationDtoIndex].locationTranslationDtoList = actionEditedLocation.location;
+            stateLocations[stateLocationIndex].locationsDto[locationDtoIndex].locationTranslationDtoList.forEach((location) => {
+              if (location.languageCode === 'ua') {
```

Comment:
use ternary operator here

Reply:
Hi, I got some issues with Github, this change is not related to this pr.